package.json

```diff
@@ -24,8 +24,7 @@
         "prepublishOnly": "npm run check-npm-publish-access; npm run tsc; npm run test; npm run lint:ci; npm run lint:styles; npm run build:prod",
         "coveralls": "cat ./coverage/lcov.info | node node_modules/.bin/coveralls",
         "release": "standard-version -t ''",
-        "prepare": "husky install",
-        "measure-enzyme-to-rtl": "node --experimental-fetch scripts/measure-enzyme-to-rtl-tests-ratio/measure-enzyme-to-rtl-tests-ratio.js"
```

Comment:
Let's also remove the script itself `scripts/measure-enzyme-to-rtl-tests-ratio/measure-enzyme-to-rtl-tests-ratio.js`

Reply:
:+1: